### ISSUE TITLE
Im Monitoring 0 Follower

### ISSUE THREAD

**Issue by moggel03** [jegade/esp8266-insta-counter#14]
Hallo, der counter hat 2 Tage super funktioniert, plötzlich wechselt er aber nicht mehr von der Uhranzeige auf die Followerzahl.
Das Monitoring zeugt immer 0 Follower an.
Hab schon alles versucht, auch komplett neue Hardware verwendet und auch in 2 unterschiedlichen Netzwerken. Internetverbindung steht Uhranzeige stimmt auch.
Irgendeine Idee?

**Comment by jfnk324** [comment]
Wenn du ihn neu geflasht hast, hast du dann auch deinen Instagram Name richtig eingegeben?

**Comment by moggel03** [comment]
Ja habe meinen Namen überprüft und auch testweise andere Namen verwendet , leider immer das selbe Ergebniss, habe auch schon geschaut ob dir Bibliothek alles aktuell ist

Holen Sie sich Outlook für Android<https://aka.ms/ghei36>

________________________________

**Comment by moggel03** [comment]
Wenn ich im Browser nur den counter einstelle und dann neu starte, wird dauerhaft starte auf dem Pannel angezeigt.
Wechsle ich auf Uhr und counter, wird dauerhaft die richtige Uhrzeit angezeigt

**Comment by jegade** [comment]
Hallo Moggel,

hast du etwas am Code geändert? Ggf. die Abfrage-Frequenz angepasst? Instagram reagiert etwas empfindlich, wenn man zu häufig Abfragen macht. Auch sollte man dann einfach den Counter mal für einen ganzen Tag auslassen, dann hat man die Chance, das man wieder Instaabfragen machen kann.

Beste Grüße
Jens

**Comment by moggel03** [comment]
Hallo, nein der Code ist unverändert, ich werde den Counter mal ausschalten und es mal testen.
Danke für die Hilfe

Holen Sie sich Outlook für Android<https://aka.ms/ghei36>

________________________________

**Comment by bensom111** [comment]
Selbes Problem bei mir.
Gestern hat es für ein paar Stunden super funktioniert. Wollte dann bezüglich der Matrix am Code spielen und hab dann letztlich wieder die Originale ino drauf - nun kommt IMMER 0....
Jemand ne Idee?

**Comment by Eisbaeeer** [comment]
@bensom111 Entweder warten, bis dein Internetrouter eine Zwangstrennung gemacht hat, oder einmal selbst Hand anlegen und die Internetverbindung neu aufbauen.

**Comment by mikus2604** [comment]
you've done a great job with this counter! but unfortunately, the same thing is happening to me. I have seen the follower count but then decided to change the refresh frequency. It only shows "0" now for any account name that I specify. Is it possible that Insta has blocked my IP temporarily?

**Comment by jegade** [comment]
yes, if you increase the refresh frequency or restart to often, insta blocked you. At the moment there is no fix for this case. Maybe some time in the far future i will provide a update with a new backend.

**Comment by mikus2604** [comment]
Thanks!

How do I change the time area? I'm in the UK and need to amend the hour -1


On 9 Apr 2021 at 12:17, Jens Gassmann ***@***.***> wrote:

yes, if you increase the refresh frequency or restart to often, insta
blocked you. At the moment there is no fix for this case. Maybe some time
in the far future i will provide a update with a new backend.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/jegade/followercounter/issues/14#issuecomment-816611396>,
or unsubscribe
